.travis.yml

```diff
@@ -0,0 +1,19 @@
+sudo: required
+
+services:
+  - docker
+
+langauge: go
+
+go:
+  - 1.14.x
```

Comment:
This should probably be updated to `1.16.x`, that's what is currently being used in the docker build.

Reply:
Ah, if it's causing issues then revert it for now, but I'll make a card to look at it again, it's an inherited problem.